

<!doctype html>
<html lang="en">
  <head>
    <!-- Required meta tags -->
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="image" property="og:image" content="https://www.shouvikmani.com/images/half_marathon_2.jpg">

    <!-- Bootstrap CSS -->
    <link href="/assets/css/bootstrap.min.css" rel="stylesheet">
    <link rel="icon" href="images/favicon.png">

    <!-- My CSS-->
    <link href="/assets/css/main.css" rel="stylesheet" type="text/css">

    <title>Xavi Loinaz</title>
  </head>
  <body>

    <br><br><br>

    <div class="row">

        <div class="col-sm-6">
            <h1>Xavi Loinaz</h1>

            <p>
                <a href="mailto:xloinaz@broadinstitute.org">Email</a>
                &middot;
                <!---
                <a href="https://twitter.com/_______" target="_blank">@_______</a>
                &middot;
                -->
                <a href="https://www.linkedin.com/in/xavi-loinaz-8562aa304/" target="_blank">LinkedIn</a>
                &middot;
                <a href="https://scholar.google.com/citations?user=GMZLNoEAAAAJ" target="_blank">Google Scholar</a>
                &middot;
                <a href="https://github.com/xaviloinaz" target="_blank">GitHub</a>
                <!---
		&middot;
                <a href="/assets/files/Xavi_Loinaz_CV_2024-12-03.pdf" target="_blank">CV</a>
		-->
            </p>

            <br>

            <p>
                Hi there! I'm an associate computational biologist at the Broad Institute of MIT and Harvard and Massachusetts General Hospital where I'm really grateful to be co-mentored by Profs. <a href="https://getzlab.org/">Gad Getz</a> and <a href="https://www.massgeneral.org/cancer-center/clinical-trials-and-research/center-for-cancer-research/investigators/rheinbay-lab">Esther Rheinbay</a>. My work currently focuses on computational cancer genomics, with a special focus on structural variation. Previously, I studied Computational Biology and Engineering at Brown, where I'm very lucky to have been advised by Prof. <a href="https://rsinghlab.org/">Ritambhara Singh</a>. 
            </p>

            <p>
                I'm also currently applying for PhD programs where I have the support of the <a href="https://new.nsf.gov/cise/graduate-fellowships">NSF CSGrad4US fellowship</a> ($159,000 total)!
            </p>

        </div>

        <div class="col-sm-6">
            <img src="/assets/images/ES2A7041_cropped_1.jpg" class="my-image" style="width: 350px;">
        </div>

    </div>

    <br><br>

    <h2>Selected Publications and Posters</h2>
    <br>

    <h5><a href="/assets/files/Broad_Retreat_Poster_2023.pdf">SSVGAR: A computational method to discover candidate structural variant driver genes based on cohorts of tumor genomes</a></h5>
    <b>Xavi Loinaz</b>, Johnathan Dagan, Chip Stewart, Jean-Baptiste Alberge, Julian Hess, Esther Rheinbay, Gad Getz
    <br>
    <i>Poster for Broad Institute Scientific Retreat, December 2023; in preparation for initial submission to Nature Methods with preprint imminent for bioRxiv</i>
    <img src="/assets/images/SSVGAR_CoMut_like_plot_without_title.png" class="publication-image">
    <i>
        SSVGAR (Significant Structural Variant Genic Association Revelation, pronounced "sugar") is a computational method to infer structural variant (SV) driver genes in cancer genome cohorts by analyzing the enrichment of putative functional effects based off breakpoint arrangements for specific genes. This is an alternative approach to traditional approaches focusing on SV breakpoint frequencies, and it can potentially be used to complement such traditional methods in the future. SSVGAR has demonstrated efficacy in recapitulating known drivers and discovering novel candidates, and was applied in a Nature Genetics paper that was accepted in principle (see two publications below).  
    </i>
    <br><br><br>

    <h5><a href="https://www.sciencedirect.com/science/article/pii/S0006497124037662">Genomic landscape of multiple myeloma and of its precursor conditions, and its clinical implications</a></h5>
    Jean-Baptiste Alberge*, Ankit K. Dutta*, Andrea Poletti*, Tim H. H. Coorens, Elizabeth D. Lightbody, Rosa Toenges, <b>Xavi Loinaz</b>, Sofia Wallin, Andrew Dunford, Oliver Priebe, Cody J. Boehner, Erica Horowitz, Nang K. Su, Hadley Barr, Laura Hevenor, Katherine Towle, Rashmika Beesam, Jenna B. Beckwith, Jacqueline Perry, David M. Cordas dos Santos, Luca Bertamini, Patricia T. Greipp, Kirsten Kübler, Peter F. Arndt, Carolina Terragna, Elena Zamagni, Eileen M. Boyle, Kwee Yong, Meletios Athanasios Dimopoulos, Efstathios Kastritis, Julian Hess, Romanos Sklavenitis-Pistofidis, Chip Stewart, Gad Getz**, Irene M. Ghobrial**
    <br>
    <i>Oral presentation at ASH, December 2024; accepted in principle to Nature Genetics</i><br>
    <a href="https://www.sciencedirect.com/science/article/pii/S000649712404641X">[Poster abstract at ASH 2024]</a><br>
    <a href="https://www.sciencedirect.com/science/article/abs/pii/S0006497123107440">[Poster abstract at ASH 2023]</a>
    <figure class="publication-image-with-caption">
        <img src="/assets/images/MM_graphic.jpg" alt="Description of image" class="publication-image" />
        <figcaption class="caption">Image taken from Mission Bio website</figcaption>
    </figure>
    <i>
        This study presents a comprehensive genomic analysis of multiple myeloma (MM) and its precursor stages, identifying candidate drivers from mutations, copy number alterations, and structural variants from 1,030 patients. Using these findings, an “MM-like” score was developed that predicts disease progression, risk classification, and subclonal dynamics, offering insights to improve early intervention and monitoring strategies. The method I developed, SSVGAR (see one above) was used to infer structural variant drivers integrated into this MM-like score, and we also found specific mutational aetiologies associated with such SV candidate drivers.
    </i>
    <br><br><br>

    <h5><a href="https://www.cell.com/cancer-cell/fulltext/S1535-6108(24)00477-X">Classification of non-TCGA Cancer Samples to TCGA Molecular Subtypes Using Compact Feature Sets</a></h5>
    Kyle Ellrott*, Christopher K. Wong*, Christina Yau*, Mauro A. A. Castro*, Jordan A. Lee*, Brian J. Karlberg*, Jasleen K. Grewal*, Vincenzo Lagani*, Bahar Tercan*, Verena Friedl, Toshinori Hinoue, Vladislav Uzunangelov, Lindsay Westlake, <b>Xavi Loinaz</b>, Ina Felau, Peggy I. Wang, Anab Kemal, Samantha J. Caesar-Johnson, Ilya Shmulevich, Alexander J. Lazar, Ioannis Tsamardinos, Katherine A. Hoadley, The Cancer Genome Atlas Analysis Network, A. Gordon Robertson, Theo A. Knijnenburg, Christopher C. Benz, Joshua M. Stuart, Jean C. Zenklusen, Andrew D. Cherniack**, Peter W. Laird**
    <br>
    <i>Published in Cancer Cell, January 2025</i><br>
    <a href="https://aacrjournals.org/cancerres/article/84/6_Supplement/6548/737566">[Oral abstract at AACR 2024]</a>
    <img src="/assets/images/CCG_TMP_graphic.png" class="publication-image">
    <i>
        This paper develops machine learning models to classify new cancer samples into predefined TCGA molecular subtypes using multi-omic data from 8,791 tumor samples across 26 cancer cohorts. By validating the models with external datasets and providing containerized versions of the top-performing classifiers, this work attemps to enable more accessible and accurate application of molecular subtyping in clinical and research settings.
    </i>
    <br><br><br>

    <h5><a href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2021.0316">Integrating Long-Range Regulatory Interactions to Predict Gene ExPression Using Graph Convolutional Networks</a></h5>
    Jeremy Bigness, <b>Xavi Loinaz</b>, Shalin Patel, Erica Larschan, Ritambhara Singh
    <br>
    <i>Published in the Journal of Computational Biology, May 2022</i>
    <img src="/assets/images/GC_MERGE_graphic.png" class="publication-image">
    <i>
        Introduces GC-MERGE, a graph convolutional model that integrates long-range genomic interactions and local regulatory factors (specifically histone modifications) to predict gene expression while capturing the 3D structure of the genome. By applying the model to multiple cell lines, the study demonstrates state-of-the-art predictive performance and interprets the biological mechanisms behind its predictions, offering potential insights into the local and long-range regulatory factors influencing gene expression.
    </i>
    <br><br><br>

    <h5><a href="https://ccmb.brown.edu/sites/default/files/Loinaz.pdf">Exploring Graph-Based Neural Networks for Modeling Long-Range Epigenetic Gene Regulation</a></h5>
    <b>Xavi Loinaz</b> (advised by Ritambhara Singh)
    <br>
    <i>Brown University Honors Senior Thesis, May 2021</i><br>
    <a href="/assets/files/Brown_2021_CS_Research_Symposium_Presentation.pdf">[Presentation for Brown CS Research Symposium]</a>
    <img src="/assets/images/Brown_senior_thesis_graphic.png" class="publication-image">
    <i>
      Introduces XL-MERGE, a modified version of GC-MERGE with three different mechanisms for improvement, outperforming GC-MERGE for all three tested cell lines in terms of AUROC. 
    </i>
    <br><br><br>

    <h5><a href="/assets/files/Xavi_Poster-Final_Colvin_Lab_2019.pdf">Polymeric Nanoparticle Surface Coatings for Biomedical Applications</a></h5>
    <b>Xavi Loinaz</b>, Xiaoting Guo, Jake Villanova, Hyewon Kim, Vicki Colvin
    <br>
    <i>Poster for Brown Undergraduate Summer Research Symposium, August 2019</i>
    <img src="/assets/images/Colvin_Lab_PAMPS_LA_Synthesis_graphic.png" class="publication-image">
    <i>
        I worked on two different research projects: one observing protein crystallization in the presence of gold nanoparticles, and one on creating gadolinium-oxide MRI contrast agents for characterizing glioma. For the latter, I determined a way to synthesize and demonstrate desired PAMPS-LA polymer surface coating of desired size, an issue that had been unresolved in the lab for approximately a year.
    </i>
    <br><br><br>

    <h5><a href="https://community.fangraphs.com/a-brief-analysis-of-predictive-pitching-metrics/">A Brief Analysis of Predictive Pitching Metrics</a></h5>
    <b>Xavi Loinaz</b>
    <br>
    <i>Published on FanGraphs Community Research Blog, January 2018</i>
    <img src="/assets/images/FanGraphs_article_graphic_2.png" class="publication-image">
    <i>
      A brief article I wrote on evaluating different Sabermetric pitching statistics for predicting future pitcher performance.
    </i>
    <br><br><br>

    <h5><a href="/assets/files/DBS_AAR_Xavi_Loinaz_Poster_2017.pdf">The Design of Visual Recognition Software to Analyze Deep Brain Stimulation Video</a></h5>
    <b>Xavi Loinaz</b>, Eric Sabelman
    <br>
    <i>Poster for Advanced Authentic Research Showcase, May 2017</i>
    <img src="/assets/images/AAR_project_graphic.png" class="publication-image">
    <i>
      Back in high school, I was a research intern in the neurosurgery department of Kaiser Permanente at Redwood City, California. I developed software to automatically parse video of deep brain stimulation based off the importance of certain parts of the surgical procedure. The image above is a snpashot of the type of video content I worked with.
    </i>
    <br><br><br>

    <h2>Selected Other Projects</h2>  
    <br>

    <h5>Characterization of somatic events for diffuse large B-cell lymphoma in the Clinical Trial Sequencing Project</h5>
    As the Getz Lab's lead analyst of the NIH-funded Clinical Trial Sequencing Project (CTSP) on diffuse large B-cell lymphoma, I fixed pipelines and analyzed somatic mutations, indels, copy number alterations, structural variants, and mutational signatures in 138 diffuse large B-cell lymphoma (DLBCL) samples. Had poster presentation for 2021 Broad Retreat Scientific Retreat; content is unable to be posted due to consortium rules. Project is progressing towards a manuscript.
    <br><br>

    <h5>Characterization and concordance analysis for cell lines of the Human Cancer Models Initiative</h5>
    Acted as the Getz Lab's lead analyst for the NIH-funded Human Cancer Models Initiative to develop next-generation cancer models that better represent the diversity and complexity of human cancers. I focused on analysis relating to structural variation and quality control, as well as concordance of mutational signatures between tumor samples and corresponding models. Anticipated submission for flagship manuscript is by the end of January 2025. 
    <br><br>

    <h5>Getz Lab's Structural Variant-Calling Pipeline</h5>
    I rewrote and maintain the tumor structural variant-calling pipeline of the Getz Lab (50+-person lab). I also implemented a scatter/gather parallelization of the SV-calling tool SvABA, reducing SvABA’s runtime by a factor of ~10, and additionally have patched several bugs in the pipeline.
    <br><br>

    <h5><a href="/assets/files/Final_IS_Project_Report_Spring_2020.pdf">Neurophysiological Experiments Data Synchronization System</a></h5>
    For spring semester of my junior year in college, I had an independent study research project where I developed a proof-of-concept system for data synchronization between video and various electrophysiological sensors for studies in Prof. David Borton's lab at Brown. I read in and decoded LTC signals from Timecode Systems devices on an Arduino and wrote software to test synchronization. I also resolved a problem in drift between GoPro and behavioral sensor data that had affected one of the lab's projects and had been unresolved for about a year.
    <br><br>

    <h5><a href="/assets/files/ScraperFinalPoster.pdf">Are.na Recommendation System</a></h5>
    For our final project in Brown's upper-level Data Science (CSCI 1951A) course, we developed a recommendation system for the art-sharing website Are.na using singular value decomposition. 
    <br><br>

    <h5>Histological Staining Quantification</h5>
    In Prof. Jill Helms' lab at Stanford as a rising college sophomore, upon seeing how long it took to manually quantify amount of histological staining on microscope slides, I created software using OpenCV to automatically quantify staining using color thresholds. 
    <br><br>

    <h5><a href="https://www.youtube.com/watch?v=NyyMx0yzfJ4&ab_channel=XaviLoinaz">TheGunnApp</a></h5>
    Back in high school, I conceptualized and led the development and maintenance of an official school-sanctioned iOS app with useful features for my high school's community. It eventually got over 5,000 downloads and was regularly used by students, parents, and faculty. This was my first formative experience in seeing how immediate of an impact computational technologies I could develop could have on the people around me!
    <br>
    <a href="https://gunnoracle.com/8023/features/the-gunn-app-provides-campus-info/">[Press]</a>
    <br>

    <br><br>


    <!---
    <h2>Selected Posters and Presentations</h2>
    <br>

    <h5><a href="/assets/files/Broad_Retreat_Poster_2023.pdf">SVGAR_sf: A novel computational algorithm to discover driver structural variant genes within cancer patient cohorts</a></h5>
    <b>Xavi Loinaz</b>, Chip Stewart, Johnathan Dagan, Jean-Baptiste Alberge, Andrew Dunford, Julian Hess, Esther Rheinbay, Gad Getz
    <br>
    <i>Poster for Annual Broad Institute Scientific Retreat, December 2023</i>
    <br><br>

    <h5><a href="/assets/files/Broad_Retreat_Poster_2021.pdf">Characterization of somatic events for diffuse large B-cell lymphoma in the Clinical Trial Sequencing Project</a></h5>
    <b>Xavi Loinaz</b>, Qing Zhang, Julian Hess, Jialin Ma, David Heiman, Chip Stewart, Gad Getz
    <br>
    <i>Poster for Annual Broad Institute Scientific Retreat, December 2021</i>
    <br><br>

    <h5><a href="/assets/files/Brown_2021_CS_Research_Symposium_Presentation.pdf">Graph-Based Neural Networks for Modeling Long-Range Epigenetic Regulation</a></h5>
    <b>Xavi Loinaz</b>, Jeremy Bigness, Shalin Patel, Erica Larschan, Ritambhara Singh
    <br>
    <i>Presentation for Annual Brown CS Research Symposium, May 2021</i>
    <br><br>

    <h5><a href="/assets/files/Xavi_Poster-Final_Colvin_Lab_2019.pdf">Optimizing PAMPS-LA Polymer Chain Size for Gadolinium-Oxide Nanoparticle MRI Contrast Agents for Detecting Glioma</a></h5>
    <b>Xavi Loinaz</b>, Xiaoting Guo, Jake Villanova, Hyewon Kim, Vicki Colvin
    <br>
    <i>Poster for Brown Undergraduate Summer Research Symposium, August 2019</i>
    <br><br>

    <h5><a href="/assets/files/DBS_AAR_Xavi_Loinaz_Poster_2017.pdf">The Design of Visual Recognition Software to Analyze Deep Brain Stimulation Video</a></h5>
    <b>Xavi Loinaz</b>, Eric Sabelman
    <br>
    <i>Poster for Advanced Authentic Research Symposium, May 2017</i>
    <br>

    <br><br>
    -->


    <h2>Mentorship and Outreach</h2>
    <br>
    <p>
        I love mentoring students and building both their enthusiasm and skills in research and academics! I have mentored three undergraduate students in Getz Lab, and used to be a classroom leader for the after-school volunteering program <a href="https://buddies4math.org/">Buddies4Math</a>.
    </p>
    <head>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <title>Image Example</title>
        <style>
            .image-container {
            text-align: center; /* Centers content horizontally */
            }
            .image-container img {
            width: 700px; /* Set the desired width */
            height: auto; /* Maintain aspect ratio */
            }
            .caption {
            font-style: italic; /* Make text italicized */
            }
        </style>
    </head>
    <body>
    <div class="image-container">
        <img src="/assets/images/IMG_1511.jpg" alt="Me with Johnathan at the poster showcase">
        <div class="caption">Me with my undergraduate intern, Johnathan, at the Getz Lab's end-of-summer intern poster showcase!</div>
    </div>
    </body>
    <br>
        Additionally, I was part of the steering committee for RATalks, a group at the Broad Institute that hosts talks to the greater community. For this, I conceptualized and led planning of our very first <a href="/assets/files/RATalk_flyer.pdf">alumni RATalk</a> which sought to disseminate career and life perspective to the early-researcher community by a former associate director of the Broad Institute, increasing the availability of mentorship to junior researchers!
    <br>
    <br><br>


    <h2>Reading and Writing</h2>
    <br>
    I love to read and average around 30 books per year. What I love most about reading is how it enables better understanding of those around us and their perspectives. I think this is especially important in research, where this can help me to be a better collaborator and also to create a more welcoming environment to others from diverse backgrounds. Some books that have inspired me:
    <p>
    <ul>
        <li><i>Free Food for Millionaires</i> by Min Jin Lee</li>
        <li><i>Everything I Never Told You</i> by Celeste Ng</li>
        <li><i>Little Failure</i> by Gary Shteyngart</li>
        <li><i>Deep Work: Rules for Focused Success in a Distracted World</i> by Cal Newport</li>
        <li><i>Miracle Creek</i> by Angie Kim</li>
        <li><i>How to Know a Person: The Art of Seeing Others Deeply and Being Deeply Seen</i> by David Brooks</li>
        <li><i>Hunger: A Memoir of (My) Body</i> by Roxane Gay</li>
        <li><i>Born a Crime: Stories From a South African Childhood</i> by Trevor Noah</li>
    </ul>
    </p>
    Due in part to the perspectives I gained from my reading, I also wrote an <a href="https://tinyurl.com/xavi-nsf-csgrad4us-essay">essay</a> about how to increase diversity in computer science-related disciplines, which incidentally helped earn me $159,000 of funding for my PhD via the <a href="https://new.nsf.gov/cise/graduate-fellowships">NSF's CSGrad4US fellowship</a>. 
    <br>

    

    <br><br>
    <div style="text-align:center; font-size:small;">
        This website's design was based on a template by <a href="https://shouvikmani.com" target="_blank">Shouvik Mani</a>.
    </div>